Umbraco-Cloud/Databases/Backups/index.md

```diff
@@ -0,0 +1,62 @@
+## Backups
```

Comment:
This is the first heading on the page, right?
Then Heading 1 should be used. I also added "Database" to the heading.
Perhaps the file name should be changes to "Database Backups" as well? 🤔

```suggestion
# Database backups
```